app/autoupdater.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const os = require('os');
+const { app, autoUpdater, dialog } = require('electron');
+const version = app.getVersion();
+const platform = os.platform() + '_' + os.arch();  // usually returns darwin_64
+const config = require('./config');
+const updaterServerUrl = config.updateEndpoint;
+
+const updaterFeedURL = `${updaterServerUrl}${platform}/${version}`;
```

Comment:
I need to give you the full URL for our feed @anri-1151 - I'll do that by tomorrow.